### ISSUE TITLE
How to add aria-label to noUiHandler

### ISSUE THREAD

**Issue by akinhwan** [leongersen/noUiSlider#1019]
How can i add an aria-label attribute to this dynamic div?

![image](https://user-images.githubusercontent.com/10124722/65161447-99b55400-da05-11e9-9042-0853fc4b6f0a.png)

**Comment by leongersen** [comment]
Sorry, what are you asking?

**Comment by akinhwan** [comment]
I ended up adding the aria-label attribute via javascript, sorry thanks. in case someone else needs to resolve axe violation for that

**Comment by toniheikkala** [comment]
I'm also doing accessibility testing with [axe](https://www.deque.com/axe/) accessibility testing tool and came across this same problem.
Like issue opener's screenshot shows, axe reports following problem: https://dequeuniversity.com/rules/axe/3.3/aria-input-field-name

Basically it would be nice if there was an option to give the slider handle "aria-label" attribute. Currently screen readers read out only the "aria-valuetext" when handle is focused, which by itself might be confusing for some users.

Other options could be giving the handle either "id" or "aria-labelledby" attribute, but these would require always having either label or some other element containing description text. [MDN examples](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role)

This is not that big a deal though, since like akinhwan said the attribute(s) can be added after slider initialization using some JS. Just thought I'd clarify what is asked here :)

**Comment by leongersen** [comment]
I'm open for suggestions on this feature. It could look something like this:

```JavaScript
noUiSlider.create(slider, {
    start: [20, 80],
    handleAttributes: [
        { 'aria-label': 'lower' },
        { 'aria-label': 'upper' },
    ],
    range: {
        'min': 0,
        'max': 100
    }
});
```

**Comment by dippas** [comment]
@leongersen  your solutions seems good to me, are you thinking implementing this ?

**Comment by leongersen** [comment]
The [`handleAttributes` option](https://refreshless.com/nouislider/slider-options/#section-handle-attributes) is now available in noUiSlider 15.4.0.

Thank you for contributing, everybody!

**Comment by jurajk** [comment]
> The [`handleAttributes` option](https://refreshless.com/nouislider/slider-options/#section-handle-attributes) is now available in noUiSlider 15.4.0.
> 
> Thank you for contributing, everybody!

Thank you @leongersen, I just wanted to see if i could implement this and possibly open a PR only to find out you already did this. Much appreciated!
